src/pyproject.toml

```diff
@@ -44,7 +46,8 @@ disable = [
   "unnecessary-lambda-assignment",
   "use-dict-literal",
 ]
-extension-pkg-allow-list = ["lxml"]
-recursive = true
+enable = [
+  "useless-suppression",
+]
```

Comment:
I based these changes on the output of `pylint --generate-toml-config`, which emits a config template showing all the supported sections, options, etc.